js/templates/pageNav.html

```diff
@@ -16,30 +16,30 @@
       </div>
       <div>
         <div class="flexVCent">
-          <a class="iconBtn ion-chevron-left js-navBack"></a>
+          <a class="iconBtn margLSm ion-chevron-left js-navBack"></a>
           <a class="iconBtn ion-chevron-right js-navFwd"></a>
-          <a class="iconBtn ion-refresh js-navReload"></a>
+          <a class="iconBtn ion-refresh js-navReload margRSm"></a>
```

Comment:
There's a margLSm class on the first link, and a margRSm class on the last link.

Adding the same amount of padding to the right and left side of the container they're in would do the same thing, and be a little cleaner (if we ever add another link, or change the order, it would still work).